manifests/cirunner.pp

```diff
@@ -34,52 +34,36 @@
 # Copyright 2015 Tobias Brunner, VSHN AG
 #
 class gitlab::cirunner (
-  $concurrent = undef,
-  $metrics_server = undef,
-  $hiera_default_config_key = 'gitlab_ci_runners_defaults',
-  $hiera_runners_key = 'gitlab_ci_runners',
-  $manage_docker = true,
-  $manage_repo = true,
-  $xz_package_name = 'xz-utils',
-  $package_ensure = installed,
-  $package_name = 'gitlab-runner',
+  Optional[Integer]       $concurrent           = undef,
+  Optional[String]        $metrics_server       = undef,
+  Boolean                 $manage_repo          = true,
+  String                  $conf_file            = '/etc/gitlab-runner/config.toml',
+  Enum[installed, absent] $package_ensure       = installed,
+  String                  $package_name         = 'gitlab-runner',
+  Optional[String]        $default_token        = undef,
+  String                  $default_url          = 'https://gitlab.com',
+  Boolean                 $default_run_untagged = true,
+  Boolean                 $default_locked       = false,
+  Optional[Array[String]] $default_tags         = undef,
+  Hash                    $docker_runners       = {},
+  Hash                    $shell_runners        = {},
+  Hash                    $ssh_runners          = {},
+  Hash                    $docker_ssh_runners   = {},
+  Hash                    $parallels_runners    = {},
+  Hash                    $virtualbox_runners   = {},
+  Hash                    $kubernetes_runners   = {},
 ) {
 
-  validate_string($hiera_default_config_key)
-  validate_string($hiera_runners_key)
-  validate_bool($manage_docker)
-  validate_bool($manage_repo)
-  validate_string($xz_package_name)
-  validate_string($package_ensure)
-  validate_string($package_name)
-
   unless ($::osfamily == 'Debian' or $::osfamily == 'RedHat')  {
     fail ("OS family ${::osfamily} is not supported. Only Debian and Redhat is suppported.")
   }
 
-  if $manage_docker {
-    include ::docker
-    # workaround for cirunner issue #1617
-    # https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/issues/1617
-    ensure_packages($xz_package_name)
-
-    $docker_images = {
-      ubuntu_trusty => {
-        image => 'ubuntu',
-        image_tag => 'trusty',
-      },
-    }
-    class { '::docker::images':
-      images => $docker_images,
-    }
-  }
-
   if $manage_repo {
     $repo_base_url = 'https://packages.gitlab.com'
 
     case $::osfamily {
       'Debian': {
-        include apt
+        include ::apt
```

Comment:
Why was this changed? using topscope isn't needed for Puppet4 and newer, in our review guidelines we agreed on not doing this. https://voxpupuli.org/docs/#reviewing-a-module-pr

Reply:
reverted back